### ISSUE TITLE
"Login" is misleading

### ISSUE THREAD

**Issue by leonissenbaum** [GRA0007/crab.fit#314]
Crab fit works great! However, I've experienced quite a few people see the website, see "Sign in to add your availability", and not bothering since they need to create an account, which they don't. To fix this, it may make sense to change the text to something along the lines of "Enter your name to add your availability", and changing the login button to "Set name".

**Comment by chris2fr** [comment]
I agree with this. It would be useful to have "Name" on a page without the crabfit, or somehow have the crabfit "greyed out" on the login part of the app
